Sources/Filters/Cornerstone/ImageDataToCornerstoneImage/index.js

```diff
@@ -37,11 +37,11 @@ function vtkImageDataToCornerstoneImage(publicAPI, model) {
     // FIXME probably need to expand to RGBA
     let pixelData = null;
     if (dims[2] === 1) {
-      pixelData = scalars.data;
+      pixelData = !scalars.data ? rawData : scalars.data;
     } else {
       const offset =
         model.sliceIndex * dims[0] * dims[1] * rawData.BYTES_PER_ELEMENT;
-      pixelData = new macro.TYPED_ARRAYS[(scalars.getDataType())](
+      pixelData = new macro.TYPED_ARRAYS[scalars.getDataType()](
```

Comment:
Try running `npm run reformat` again. The style linter is complaining about this line, and is asking for you to surround the property access like so: `[(scalars.getDataType())]`.